pandas/core/arrays/datetimes.py

```diff
@@ -82,7 +82,7 @@ def f(self):
         return result
 
     f.__name__ = name
-    f.__doc__ = "\n{}\n".format(docstring)
+    f.__doc__ = "{}".format(docstring)
```

Comment:
What is this change for?

Reply:
@WillAyd For some reason I am not able to push my commits. I don't know why. Here is the error message.
![image](https://user-images.githubusercontent.com/25010571/49783198-08cb8280-fce7-11e8-8ed9-defb648e71a8.png)

Reply:
Assuming origin is your own repo and you are working from a different computer than before you are behind your repo locally. Quite a few ways to solve; depending on how much work you just did easiest may be to reset to the prior commit, pull from master than redo your change on top. Rebasing is another option

Reply:
I tried to revert but still unable to push. Not too sure what to do at this point.

Reply:
```
f.__doc__ = docstring
```
but before, can you run the validation script for one of the functions generated with this, and paste the output in a comment?

I think this was required to avoid the error of having the opening quotes in the same line as the summary. Is this change required for the validation, or ut was just to clean up like in the other cases where `\n` is removed?